src/delay.js

```diff
@@ -67,17 +67,15 @@ define(function (require) {
      *  <a href="http://www.w3.org/TR/webaudio/#DelayNode">
      *  Web Audio Delay Nodes</a>, one for each stereo channel.
      *
-     *  @property leftDelay
-     *  @type {Object}  Web Audio Delay Node
+     *  @property leftDelay {Object}  Web Audio Delay Node
```

Comment:
i would definitely keep the property names. let's see if we can't fix the template so they don't appear where they're not wanted.